### DISCUSSION TITLE
Cinelerra

### DISCUSSION THREAD

**Discussion by JuliaAgostino** [mayrinck/FOSSonMicrosoftStore#17] [Requests] [answered]
Cinelerra website: http://www.cinelerra.org/

Cinelerra privacy policy: https://www.iubenda.com/privacy-policy/92641166/full-legal

Logo: https://upload.wikimedia.org/wikipedia/commons/5/51/Cinelerra.svg

Description: "Cinelerra is a video editing and composition program (an NLE, Non-Linear Editor) designed for the Linux operating system. It is free software distributed under the open source GNU General Public License. In addition to editing, it supports advanced composition operations such as keying and mattes, including a title generator, many effects to edit video and audio, keyframe automation, and many other professional functions depending on the variant. It processes audio in 64 floating-point form. Video is processed in RGBA or YUVA color spaces, in 16-bit integer or floating-point form. It is resolution and image refresh rate independent, which means it can support video of any speed and size; this applies to the HV variant, the GG variant can support up to 8K video. The GG variant can also create DVDs and Blu-rays."

**Comment by mayrinck** [comment]
I was preparing to convert this one but I couldn't find a Windows version for this one... Is there a link or something?

**Reply by JuliaAgostino** [reply]
> I was preparing to convert this one but I couldn't find a Windows version for this one... Is there a link or something?

A website I was reading claims that there's a version for Windows, OS X, and Linux. That's all I could go off of. 🤷🏼‍♂️

**Comment by mayrinck** [comment, marked as answer]
Well, I couldn't find a package to convert for this one so... I will skip it, for now... Maybe later we can come back to this one 😃
